.github/workflows/ci.yml

```diff
@@ -44,6 +44,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: fastify/github-action-merge-dependabot@v2.0.0
-        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
```

Comment:
hey @Fdawgs If I remember correctly we added this because we were not running the checks on the pull_request.

Reply:
it's not needde anymore

Reply:
Hi @aboutlo, as @mcollina said, with v2.0.0 of the action it handles all this for us. So the 'if' conditional is no longer needed.